package.json

```diff
@@ -20,6 +20,7 @@
         "draft",
         "azure"
     ],
+    "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
```

Comment:
is this a test key or the final production key?

Reply:
It's a production key, but it seems that other open source Microsoft extensions do the same thing (e.g. https://github.com/Microsoft/vscode-docker/blob/master/package.json).  It gave me the jitters too, but I don't see a way of concealing it because even if we added it as part of a publish process it would still be in the `package.json` in the VSIX.

Reply:
got it.